app/models/public_comment.rb

```diff
@@ -1,30 +1,32 @@
 class PublicComment < Comment
 
-  after_create :create_notifications, :send_emails
+  after_create :notify, :send_emails
 
   private
 
   # Send emails to speakers when reviewer creates a comment
   def send_emails
     if user.reviewer_for_event?(proposal.event)
```

Comment:
As this check is already done, move the two mailer calls to the same section in notify.  Then add users to the params list since you've already constructed those lists there.